app/controllers/posts_controller.rb

```diff
@@ -116,7 +116,9 @@ def post_event
     end
 
     def send_slack
-      notifier = Slack::Notifier.new Rails.application.secrets.slack_url
-      notifier.post unfurl_links: true, unfurl_media: true, text: "New post published: *[#{@post.title_sv}](#{post_url(@post, locale: 'sv')})* by #{@post.user.display_name}"
+      cthit_notifier = Slack::Notifier.new Rails.application.secrets.slack_url
```

Comment:
Shouldn't this be ``cthit_slack_url`` like in secrets.yml or vice-versa?

Reply:
Yes it should!

Reply:
Maybe extract duplicated logic to a function?